### PR TITLE
Also get access to the headers set by superagent

### PR DESCRIPTION
**Pull request by oponder**
For my tests I also needed access to the headers set by the 'set' method.

This is how I resolved that, but it might not be the most correct way?

If you think this feature is nice, but the implementation isn't feel free to suggest a more correct implementation, I'll make sure it works as you suggest then!

### CODE REVIEW

**Comment by fdubost** [comment]
Give the headers to the fixture method might be the good way :smiley_cat: :v: 
Could add tests and update README with this feature please ?

File: lib/superagent-mock.js [hunk 1]
```diff
@@ -54,6 +54,14 @@ function mock (superagent, config) {
   };
 
   /**
+   * Override set function
+   */
+  Request.prototype.set = function (headers) {
+    this.headers = headers
+    return this;
```
Comment (fdubost) [inline lib/superagent-mock.js]: Put a blank line before `return` statement :wink:

**Comment by oponder** [comment]
Thanks for the fast review! Working on readme and tests.

Not sure what you mean with 'Give the headers to the fixture method'
Is that not what I am doing? Tips are appreciated!

**Comment by fdubost** [comment]
Yes, it's what you're doing, it's good for me :wink:

**Comment by oponder** [comment]
@fdubost: ready for another review :D

**Comment by fdubost** [comment]
It seems to be OK for me :+1: 
Thank you @oponder :smiley:

**Comment by fdubost** [comment]
Tag v1.5.0. I can't publish this on npm now because only @oziks have the rights and he is on holiday :smile_cat:

**Comment by oponder** [comment]
thats ok :) take your time

**Comment by fdubost** [comment]
Available on npm under 1.7.0 version.

**Pull request merged**